Decrypt demo secrets into DerivedData, outside the repo (#822)
Automattic/Gravatar-SDK-iOS: Sources/GravatarUI/GravatarUI.docc/GettingStarted.md

2. To have access to the Quick Editor through OAuth, set the following `Callback URI`:
  - `https://gravatar.com/iosdemo/oauth/callback` 
3. Copy the `API key`, `Client ID`, and `Callback URI` to be pasted on the demo app.

> 4. Internal contributors: run `make setup-secrets` to decrypt the demo credentials (stored outside the repo under `~/.configure`). External contributors: build the demo once — it seeds a gitignored `Demo/Demo/Secrets.external-contributors.swift` from the template — then fill in the values from the previous step there.

Author: ```suggestion
4. Internal contributors: run `make setup-secrets` to decrypt the credentials outside the repo. External contributors: build the app once and it will seed a gitignored `Demo/Demo/Secrets.external-contributors.swift` from a template, then fill in the values from the previous step there.
```
